persist/sqlite/consensus.go

```diff
@@ -296,15 +297,27 @@ func (ut *updateTx) AddSiacoinElements(elements []types.SiacoinElement, index ty
 	}
 	defer addrStmt.Close()
 
+	indexStmt, err := insertIndexStmt(ut.tx)
+	if err != nil {
+		return fmt.Errorf("failed to prepare index statement: %w", err)
+	}
+	defer indexStmt.Close()
+
 	// ignore elements already in the database.
-	insertStmt, err := ut.tx.Prepare(`INSERT INTO siacoin_elements (id, siacoin_value, merkle_proof, leaf_index, maturity_height, address_id, matured) VALUES ($1, $2, $3, $4, $5, $6, $7) ON CONFLICT (id) DO NOTHING RETURNING id`)
+	insertStmt, err := ut.tx.Prepare(`INSERT INTO siacoin_elements (id, siacoin_value, merkle_proof, leaf_index, maturity_height, address_id, matured, chain_index_id) VALUES ($1, $2, $3, $4, $5, $6, $7, $8) ON CONFLICT (id) DO NOTHING RETURNING id`)
```

Comment:
Why do you specify `RETURNING id`? Won't this return rows that `CONFLICT` on `id`, even though we can safely ignore those? We're not using `dummyID` either by the way, not sure if it's a debug left over bug figured I'd mention it.

Reply:
It will not. If there’s a conflict, the `RETURNING` clause is dropped. As far as the dummyID, it’s usually in places where I want to ensure that something happened or do a sanity check on the database.